frontend/pages/docs/users/v-two-docs/creating-a-model.mdx

```diff
@@ -0,0 +1,53 @@
+import Image from 'next/legacy/image'
+import Box from '@mui/material/Box'
+import bailoMarketIcon from '../../../../public/docs/bailo_market_icon.png'
+
+## **Bailo Marketplace**
+
+The models added to Bailo are displayed in the Bailo Marketplace.
+
+<Box sx={{ maxWidth: '75%', margin: 'auto' }}>
+  <Image src={bailoMarketIcon} />
+</Box>
+
+This enables the models to be discoverable to other users. There's an exception to this, where the models are marked as
+private. In these instances the model will not be displayed.
+
+## **Finding models**
+
+Models are discoverable by the search functionality or via filtering on the tags.
+
+## **Adding a model to the Marketplace**
+
+Selecting to **_Add a new model_** will enable you to create a model.
+
+Adding this to the Marketplace is quick and easy and requires 3 fields to be completed.
+
+The fields are:
+
+- **_*The Model Name and the Model Description.*_**
+
+These represent what is displayed for the model in the Marketplace.
+
+- The 3rd field that requires completion is the **_*Public*_** or **_*Private*_** option.
+
+The choice of these will determine whether model is displayed in the Marketplace.
+
+The model is now added to the Marketplace.
+
+## **Adding the Model Card detail**
+
+Once the initial 3 fields have been completed and the model is added to the Marketplace, the user is provided with the
+option to select which **_*schema*_** they wish to use.
+
+- The selection of the schema defines the model card detail that will be required for the model.
```

Comment:
"The selection of the schema defines the model card detail that will be required for the model." -> "The selection of the schema defines the model card structure that will be required for the model.